pkg/apis/pipeline/v1/result_types.go

```diff
@@ -45,16 +45,16 @@ type TaskRunResult struct {
 	Type ResultsType `json:"type,omitempty"`
 
 	// Value the given value of the result
-	Value ArrayOrString `json:"value"`
+	Value ResultValues `json:"value"`
 }
 
+type ResultValues = ParamValue
```

Comment:
can this be `ResultValue` instead of `ResultValues`?

Reply:
Sure! Thanks!